linode/manager
# M3-5183: React Query for Kubernetes - Part 1

## Description 📝

- This PR focuses getting React Query to power the Kubernetes landing page table and associated creation and deletion actions
  - > **Warning**: All other Kubernetes flows and actions will be addressed in part 2 of this PR

## Preview 📷

![Screen Shot 2022-12-14 at 7 52 51 PM](https://user-images.githubusercontent.com/115251059/207746852-b742d66d-1cf3-42bd-8334-64be735277e9.jpg)

## How to test 🧪

- Testing Kubernetes Landing
  - Test deleting a Kubernetes Cluster and verify the table updates
  - Test creating a Kubernetes Cluster and verify the table updates
